src/index.html

```diff
@@ -4,14 +4,181 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="/action_page.php" autocomplete ="off">
+      <fieldset class="InsideIndent">
+        <legend>Personal information</legend>
+        <label class="inside">
+          Surname:
+          <input
+            type="text"
+            id="fname"
+            name="camelCase"
+            required
+          >
+        </label>
+        <label class="inside">
+          Name:
+          <input
+            type="text"
+            id="lname"
+            name="camelCase"
+            required
+          >
+        </label>
+        <label class="inside">
+          How old are You?
+          <input
+            type="number"
+            id="quantity"
+            name="camelCase"
+            min="1" max="100"
+            step="1" value="12"
+            autocomplete ="off"
+          >
```

Comment:
```suggestion
            name="camelCase"
            min="1"
            max="100"
            step="1"
            value="12"
            autocomplete ="off"
          >
```
1 attribute per line